.github/ISSUE_TEMPLATE/general_issue.md

```diff
@@ -0,0 +1,26 @@
+---
+name: General issue
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please fill out the sections below to help us address your issue.
+
+### Version of AWS SDK for PHP?
+
+
+### Version of PHP (`php -v`)?
+
+
+### What issue did you see?
+
+
+### Steps to reproduce
+If you have an runnable example, please include it.
```

Comment:
We may want to suggest linking to a repository for larger examples.  In-line snippets are good too, just don't want to encourage uploading zip files.